src/key.h

```diff
@@ -15,6 +15,15 @@
 #include <stdexcept>
 #include <vector>
 
+enum OutputType
```

Comment:
I don't think this belongs in `key.h`. `OutputType` is only ever used as part of wallet code.

Reply:
@laanwj I switched to @ryanofsky's `enum MyEnum : int` approach and moved the enum back to `wallet.h`.

Reply:
Thanks